spec/01-unit/21-dns-client/03-client_cache_spec.lua

```diff
@@ -519,9 +625,15 @@ describe("[DNS client cache]", function()
           },
         }
       }
+
       client.toip("demo.service.consul")
+
+      -- It only stores SRV target entry into cache, not storing A entry.
       local success = client.getcache():get("192.168.5.232.node.api_test.consul")
-      assert.equal(client.TYPE_A, success)
+      assert.equal(nil, success)
```

Comment:
should be `assert.is_nil()`?

Reply:
done, I have fixed all the nil matches  using `assert.is_nil()` in the latest commit